docs/readmes/howtos/troubleshooting/datapath_connectivity.md

```diff
@@ -110,13 +110,14 @@ on UE or the server (on SGi side of the network) while debugging the issue.
 5. From here onwards you are going to debug OVS datapath, so you need to select
    a UE and identify which traffic direction is broken. You can do so by
    - Generating uplink traffic in UE
-   - Capturing packets on gtp_br0 in NATed datapath case and SGi_dev interface for
-     NonNAT datapath: `tcpdump -eni gtp_br0 host $UE_IP`.
-   - For NATed datapath you also need to check if packet are egressing on
+   - Capture packets on gtpu_sys_2152 device: `tcpdump -eni gtpu_sys_2152 host $UE_IP`. If you do not see any packet, it means the packets are not reaching GTP tunnel, check S1 connectivity.
+   - *NATed datapath*: Capture packets on gtp_br0 `tcpdump -eni gtp_br0 host $UE_IP`. If you dont see any packets debug using `dp_probe_cli.py` utility. This utility would show which OVS table is dropping the packet.
+   - *NATed datapath*: you also need to check if packet are egressing on
      the SGi port. You can do so by running tcpdump on SGi port
-     `tcpdump -eni $SGi_dev dst $SERVER_IP`
-   - In case the packet is missing on SGi port, you have issue with the routing.
-     check routing table on the AGW.
+     `tcpdump -eni $SGi_dev dst $SERVER_IP`. In case the packet is missing on SGi port, you have issue with the routing. Check routing table and iptables rules on the AGW.
+   - *Non-NAT datapath*: you also need to check if packet are egressing on
+     the SGi port. You can do so by running tcpdump on SGi port
+     `tcpdump -eni $SGi_dev dst $SERVER_IP`. If you dont see any packets debug using `dp_probe_cli.py` utility. This utility would show which OVS table is dropping the packet.
```

Comment:
Duplication to the `NATed datapath` case. There should be a clearer distinction, what is applicable for both cases or only one.

Reply:
thats what the prefix (*Non-NAT datapath*, "*NATed datapath*") trying to do.

Reply:
Let me maybe be more explicit here. The text is:

>    - *NATed datapath*: you also need to check if packet are egressing on
>      the SGi port. You can do so by running tcpdump on SGi port
>      `tcpdump -eni $SGi_dev dst $SERVER_IP`. In case the packet is missing on SGi port, you have an issue with the routing. Check the routing table and iptables rules on the AGW.
>    - *Non-NAT datapath*: You also need to check if packet are egressing on
>      the SGi port. You can do so by running tcpdump on SGi port
>      `tcpdump -eni $SGi_dev dst $SERVER_IP`. If you dont see any packets, try debugging with `dp_probe_cli.py` utility. This utility would show which OVS table is dropping the packet.

More than half of it is duplicated (3 lines):
```markdown
You also need to check if packet are egressing on
the SGi port. You can do so by running tcpdump on SGi port
`tcpdump -eni $SGi_dev dst $SERVER_IP`. 
```
Imho it would be better to avoid the confusing duplication of those parts and be clearer and more structured here. E.g. one could do:
```markdown
- You also need to check if packet are egressing on the SGi port. You can do so by running tcpdump
on the SGi port `tcpdump -eni $SGi_dev dst $SERVER_IP`.
  - *NATed datapath*: In case ...
  - *Non-NAT datapath*: If you do not ...
```

Reply:
Users are either going to debug NAT or Non-NAT datapath, so duplicate is not issue here.
The suggested structure looks more confusing to user who is looking to debug either NAT ot Non_NAT datapath.

Reply:
In that case, that users look for either NAT or non_NAT debugging, we should than probably group by that aspect to make this easier for the user.

Reply:
That is worth doing it, but it would be separate task. I have created doc for this: https://github.com/magma/magma/issues/12719